src/com/leetcode/problem/easy/LengthOfLastWord.java

```diff
@@ -0,0 +1,46 @@
+package com.leetcode.problem.easy;
+
+/**
+ * Given a string s consisting of some words separated by some number of spaces, return the length of the last word in the string.
+ *
+ * A word is a maximal substring consisting of non-space characters only.
+ *
+ *
+ *
+ * Example 1:
+ *
+ * Input: s = "Hello World"
+ * Output: 5
+ * Explanation: The last word is "World" with length 5.
+ * Example 2:
+ *
+ * Input: s = "   fly me   to   the moon  "
+ * Output: 4
+ * Explanation: The last word is "moon" with length 4.
+ * Example 3:
+ *
+ * Input: s = "luffy is still joyboy"
+ * Output: 6
+ * Explanation: The last word is "joyboy" with length 6.
+ *
+ *
+ * Constraints:
+ *
+ * 1 <= s.length <= 104
+ * s consists of only English letters and spaces ' '.
+ * There will be at least one word in s.
+ */
+public class LengthOfLastWord {
+
+    public static void main(String[] args) {
+        String[] s = {"Hello World", "    ", "awesome"};
+        assert lengthOfLastWord(s[0]) == 5;
+        assert lengthOfLastWord(s[1]) == 0;
+        assert lengthOfLastWord(s[2]) == 7;
+    }
+
+    public static int lengthOfLastWord(String s) {
+        String[] words = s.split("\s+");
+        return words[words.length - 1].length();
```

Comment:
I have a feeling it might fail on trailing spaces. Check 
`s = "   fly me   to   the moon  "`

Reply:
It works well for trailing white spaces as well. How do I optimize it? My submission was just 6% faster than the others.